### PR TITLE
Better lines legends and colors

### PR DESCRIPTION
**Pull request by otfrom**

### CODE REVIEW

**Review by seb231 — COMMENTED** [review]
Can you clarify where `add-watermark` is @otfrom ?

File: src/kixi/plot/series.clj [hunk 1]
```diff
@@ -3,7 +3,7 @@
             [tablecloth.api :as tc]))
 
 (defn line-series [{:keys [ds x y color shape stroke size dash]
-                    :or {shape \O
+                    :or {;;shape \O
```
[the remark this answers is not in the document]
Comment (otfrom) [reply src/kixi/plot/series.clj:6]: I think I want to get rid of the shapes on the line only series

File: src/kixi/plot.clj [hunk 2]
```diff
@@ -112,7 +114,7 @@
                 (plotb/add-label :top (::label title) title-format)
                 (plotb/add-legend legend-label legend-spec)
                 (plotr/render-lattice size)
-                )})))
+                (add-watermark watermark))})))
```
[the remark this answers is not in the document]
Comment (otfrom) [reply src/kixi/plot.clj:117]: it is on line 20

File: src/kixi/plot/series.clj (hunk 1 above)
[the remark this answers is not in the document]
Comment (seb231) [reply src/kixi/plot/series.clj:6]: 👍

File: src/kixi/plot.clj (hunk 2 above)
[the remark this answers is not in the document]
Comment (seb231) [reply src/kixi/plot.clj:117]: ah it was already in there but not being used here 👍  I see

**Pull request merged**